Error: Cannot find module '../util/Snowflake'

**Please describe the problem you are having in as much detail as possible:**

When trying to start my bot I am getting the error
```js
bannerbomb@DESKTOP-B66Q6A0:/mnt/c/Users/Admin/Desktop/testbot$ node .
node:internal/modules/cjs/loader:928
  throw err;
  ^

Error: Cannot find module '../util/Snowflake'
Require stack:
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/Sticker.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/Message.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/managers/MessageManager.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/GuildMember.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/managers/GuildMemberManager.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/Guild.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/managers/GuildManager.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/client/Client.js
- /mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/index.js
- /mnt/c/Users/Admin/Desktop/testbot/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:997:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/Sticker.js:5:19)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:997:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/Sticker.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/Message.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/managers/MessageManager.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/interfaces/TextBasedChannel.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/GuildMember.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/managers/GuildMemberManager.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/structures/Guild.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/managers/GuildManager.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/client/Client.js',
    '/mnt/c/Users/Admin/Desktop/testbot/node_modules/discord.js/src/index.js',
    '/mnt/c/Users/Admin/Desktop/testbot/index.js'
  ]
}
```

**Reproducible code:**
```js
const { Client, Intents } = require('discord.js');

const client = new Client({
	intents: Intents.ALL
})

client.on('ready', async() => {
	console.log('READY');
});

client.login('...')
```

**Further details:**

- discord.js version: v12.5.0
- Node.js version: v15.7.0
- Operating system: Ubuntu 18.04.3 LTS
- Priority this issue should have – please be realistic and elaborate if possible: high

**Relevant client options:**

- partials: none
- gateway intents: `GUILDS, GUILD_PRESENCES, GUILD_MEMBERS, GUILD_EMOJIS, GUILD_BANS, GUILD_MESSAGES, GUILD_MESSAGE_TYPING, GUILD_MESSAGE_REACTIONS, DIRECT_MESSAGES, DIRECT_MESSAGE_TYPING, DIRECT_MESSAGE_REACTIONS`
- other: none

- I have also tested the issue on latest master, commit hash: `026afc2`

--- a/src/structures/Sticker.js
+++ b/src/structures/Sticker.js
@@ -2,7 +2,7 @@
 
 const Base = require('./Base');
 const { StickerFormatTypes } = require('../util/Constants');
-const Snowflake = require('../util/Snowflake');
+const Snowflake = require('../util/SnowflakeUtil');
 
 /**
  * Represents a Sticker.

